src/main/resources/assets/alloy_forgery/lang/zh_cn.json

```diff
@@ -0,0 +1,24 @@
+{
+  "block.alloy_forgery.bricks_forge_controller": "红砖锻造控制器",
+  "block.alloy_forgery.stone_bricks_forge_controller": "石砖锻造控制器",
+  "block.alloy_forgery.deepslate_bricks_forge_controller": "深板岩砖锻造控制器",
+  "block.alloy_forgery.end_stone_bricks_forge_controller": "末地石砖锻造控制器",
+  "block.alloy_forgery.polished_blackstone_forge_controller": "磨制黑石锻造控制器",
+  "block.alloy_forgery.prismarine_bricks_forge_controller": "海晶石砖锻造控制器",
+
+  "tooltip.alloy_forgery.forge_tier": "等级: %d",
+  "tooltip.alloy_forgery.fuel_capacity": "燃料容量: %d",
+
+  "container.alloy_forgery.forge_controller": "合金锻造器",
+
+  "container.alloy_forgery.rei.title": "锻造合金",
+  "container.alloy_forgery.rei.min_tier": "锻造器等级要求: %d",
+  "container.alloy_forgery.rei.fuel_per_tick": "燃料消耗: %d/tick",
+
+  "message.alloy_forgery.invalid_multiblock": "无效的多方块配置",
```

Comment:
`"message.alloy_forgery.invalid_multiblock": "无效的多方块配置"` is literal translation, it can also be `"message.alloy_forgery.invalid_multiblock": "无效的多方块结构"` means *Invalid Multiblock Structure*

Reply:
I would translate this like `多方块结构无效`, as an actual sentence instead of a phrase